blog/2021-11-30-Building-a-Centralized-Annotated-Database.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Building a Centralized Annotated Database
+author: Punya Modi
+description: What is the need to create a centralized annotated database? What advantages can it offer?
+tags: Machine Learning =
+---
+
+# **Value of Creating a Standardized Annotated Database**
+
+As I was learning about ML and getting introduced to the world of Machine Learning one of the more memorable statements I heard was that “Machine Learning is, after all, data driven AI, and your model will be only as good or as bad as the data you have”. ML depends heavily on data, without data, it is impossible for an “AI” to learn. It is the most crucial aspect that makes algorithm training possible… No matter how great the AI team is or the size of the data set, if your data set is not good enough, the entire project will fail. While gathering a sizable amount of data is important, it is only the first step of the entire process, every project requires a sizable amount of time and effort to restructure the data and classify and label it. This is especially true for datasets used to present a realistic vision of our world. With the need for data so clear, the need arises for a mechanism to store all of this data in a fashion that doesn't compromise on the  work that was already done to tailor the data to the needs of the project.
+
+
+# FoodDX
+
+FoodDX is a product from Holmusk that is meant to transform the way consumers think and feel about food, it gives real-time feedback on your meals, FoodDX allows users to upload photos of their meal for real-time nutrition scores and food tips. This is enabled via AI food scoring that leverages the companies real world dataset of over 10,000 + images.

```

Comment:
```suggestion
FoodDX is a product from Holmusk that is meant to transform the way consumers think and feel about food. It gives real-time feedback on your meals. FoodDX allows users to upload photos of their meal for real-time nutrition scores and food tips. This is enabled via AI food scoring that leverages real world dataset of over 10,000+ images.
```

A couple of things here:
- you could add a few more things about fooddx on what it does. Examples at the _our solutions_ section of https://fooddx.com. Preferably as bullet points.
- 10,000 is not a big dataset (which means not that valuable either). It needs to be 100,000+. (We crossed 100,000 in our first 2 weeks of launch itself)